app/controllers/application_controller.rb

```diff
@@ -14,7 +14,12 @@ def current_user
   end
 
   def logged_in?
-    current_user != nil
+    !!current_user
```

Comment:
What's the difference? Is just a matter of preference?

Reply:
Yeah this is more readable